packages/utils/src/lib/collect/index.ts

```diff
@@ -28,8 +28,10 @@ export type CollectOptions = GlobalCliArgs & CoreConfig;
  * Run audits, collect plugin output and aggregate it into a JSON object
  * @param options
  */
-export async function collect(options: CollectOptions): Promise<Report> {
-  const { version, name } = await readPackageJson();
+export async function collect(
```

Comment:
The package version is always fixed. We maybe always take the version from utils package and don't expose it to options?

I'm afraid if we expose it it get misused...

Reply:
Well, we want to use package metadata for the `cli` package, so if `utils` was in charge of importing it, then `utils` would depend on `cli`, which is a big no-no for the dependency graph :grimacing: 

The only reasonable alternative I can think of is that we remove the `package.json` logic from `utils` completely. This `collect` function would the report without the package metadata (it's optional, so our types will allow it), and the CLI command adds it there afterwards. Would do you think? :thinking:

Reply:
> and the CLI command adds it there afterwards
This sounds like a good option.